packages/core/src/auth/sso/ssoAccessTokenProvider.ts

```diff
@@ -808,7 +808,7 @@ class DiskCacheErrorMessage {
             : ToolkitPromptSettings.instance
 
         // We know 'ssoCacheError' is in all extension prompt settings
-        if (await promptSettings.isPromptEnabled('ssoCacheError')) {
+        if (promptSettings.isPromptEnabled('ssoCacheError')) {
```

Comment:
This is a welcome change 👍  
Not a strong reason for this function to be async

Reply:
isolated this change here: https://github.com/aws/aws-toolkit-vscode/pull/6255/files